manticore/platforms/evm.py

```diff
@@ -1884,16 +1896,14 @@ def __str__(self):
         def hexdump(src, length=16):
             FILTER = ''.join([(len(repr(chr(x))) == 3) and chr(x) or '.' for x in range(256)])
             lines = []
-            for c in xrange(0, len(src), length):
+            for c in range(0, len(src), length):
                 chars = src[c:c + length]
-
                 def p(x):
                     if issymbolic(x):
                         return '??'
                     else:
                         return "%02x" % x
                 hex = ' '.join([p(x) for x in chars])
```

Comment:
brackets `[...]` are redundant (as in https://github.com/trailofbits/manticore/pull/784/files?diff=unified#r175897327)

btw this overrides `hex` builtin function :(